### DISCUSSION TITLE
New tests on metadata for IACS datasets

### DISCUSSION THREAD

**Discussion by MarcoMinghini** [INSPIRE-MIF/helpdesk-validator#770] [Announcements]
A set of new tests to check conformance of metadata for IACS datasets will be included in the next release (v.2022.1) of the INSPIRE Reference Validator.

These tests, developed in the framework of the collaboration between DG ENV and DG AGRI, are aimed at providing Integrated Administration and Control Systems (IACS) data providers and paying agencies acting in the scope of the Common Agricultural Policy (CAP) with a common tool for assessing the conformity of INSPIRE agricultural dataset metadata to the IACS-specific metadata requirements stated in the [Technical Guidelines on IACS Spatial Data Sharing, Part 1 – Data discovery](https://publications.jrc.ec.europa.eu/repository/handle/JRC121450). Such requirements have to be fulfilled on top of the generic metadata requirements stated in the [INSPIRE Metadata technical guidelines](https://inspire.ec.europa.eu/id/document/tg/metadata-iso19139).

The Abstract test Suites (ATS) describing the mentioned tests can be accessed at https://github.com/inspire-eu-validation/metadata/blob/2.0/iacs/IACS-datasets.md

Conformity to this set of tests is crucial for achieving the discoverability of IACS datasets through the INSPIRE infrastructure, since the tests are checking the rules for providing interoperable IACS metadata. It is indeed the first pillar for the implementation of the future Common European Agricultural Data Space (CEADS).

**Comment by ghost** [comment]
Hi @MarcoMinghini,

it is planned to implement the tests in the staging instance before official release in the production instance? 
If yes, it would be good to inform the community in order to be able to give feedback on this. Also, it would be nice to have example metadate for this in order to help community to fix the errors.

**Comment by MarcoMinghini** [comment]
Dear @DeordD, tests are already available since the end of 2021 in the staging instance. As you can see, this is an extra test on metadata which is not selected by default, since it is not a core INSPIRE test.

The Member States paying agencies have been already informed about the new tests and have started to test their metadata and provide feedback; this is why we are confident that, with the improvements made over the last few months, the tests are now mature enough to be moved to the production instance in the upcoming release.

Everyone else is also invited to test and provide feedback. Our team would be happy to provide you with sample datasets.

**Comment by fabiovinci** [comment]
Dear @DeordD,

please find here (https://github.com/INSPIRE-MIF/helpdesk-validator/blob/master/examples/IACS_dataset_metadata_2.0_example.xml) a sample MD for IACS datasets.

**Comment by ghost** [comment]
Hi @MarcoMinghini, @fabiovinci 

thanks a lot for the explanation. 
I will take a quick look at it from the metadata editor perspective and give my feedback on it. 
Sample metadata will help a lot in oder to perform those tests.

**Comment by ghost** [comment]
Hi,

sorry, but I have one more question on this: 
- Is it allowed to add codelist values (INSPIRE-Themes, IACS) in all EU languages or only English is allowed?

Thanks!

Kind regards,
Dejan

**Comment by fabiovinci** [comment]
Dear @DeordD,

if you use the `gmx:Anchor` encoding you can use any value (e.g. InVeKoS) since the checks are done on the `href `attribute.

```xml
               <gmd:keyword>
                  <gmx:Anchor
                     xlink:href="http://inspire.ec.europa.eu/metadata-codelist/IACSData/iacs">InVeKoS</gmx:Anchor>
               </gmd:keyword>
```

In case of `gco:CharacterString` encoding, at the moment, the checks are done considering the English values.

```xml
        <gmd:keyword>
            <gco:CharacterString>IACS</gco:CharacterString>
        </gmd:keyword>
```

**Comment by ghost** [comment]
Hi @fabiovinci,

In case of  `gco:CharacterString` encoding:
- The INSPIRE directive allows to encode INSPIRE-Themes all EU languages. Conformance Class 2c (IACS Metadaten) only allows INSPIRE-Themes to be encoded in English.

It would be good to support all EU-languages for both encodings in INSPIRE-validator because translation is already done (at least for German language) and implemented in INSPIRE Registry. This was also already done for Priority Data Sets and Spatial Scope. 

Thanks a lot for the examples and the efforts done so far!
